css/_popover.scss

```diff
@@ -10,10 +10,17 @@
     right: 0;
     width: 100%;
 }
+.popover-mousemove-padding-left {
```

Comment:
Do you think it would be nice to have a popover class that contains all the shared properties, like height, position and width?